### PR TITLE
Ee 724/update ega ingestion pipeline

### PR DESCRIPTION
**Pull request by ashutoshebi**
Kindly review.

### CODE REVIEW

File: file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/persistent/repository/EncryptParametersRepository.java [hunk 1]
```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  *
  */
-package uk.ac.ebi.ega.file.encryption.processor.persistence.repository;
+package uk.ac.ebi.ega.file.encryption.processor.persistent.repository;
```
Comment (jorizci) [inline file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/persistent/repository/EncryptParametersRepository.java]: For package name it is persistence not persistent.

File: file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/services/EncryptService.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        final LocalDateTime startExecution = LocalDateTime.now();
+        try {
+            result = execute(jobExecution);
+        } catch (JobNotRegistered jobNotRegistered) {
+            result = Result.abort("Unexpected exception - JobParameters not registered", jobNotRegistered,
+                    LocalDateTime.now());
+        }
+        if (!Result.Status.SUCCESS.equals(result.getStatus())) {
+            //TODO Needs to be implemented
+            // mailingService.sendSimpleMessage(reportTo, result.getMessage(), result.getException());
```
Comment (jorizci) [inline file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/services/EncryptService.java]: You have to report to file manager on success

File: file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/services/EncryptService.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        if (!Result.Status.ABORTED.equals(result.getStatus())) {
+            // report to file manager if it has finished.
```
Comment (jorizci) [inline file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/services/EncryptService.java]: If we had to do a full abort then we need re receive a warning ourselves

File: file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/services/EncryptService.java (hunk 2 above)
Comment (ashutoshebi) [reply file-encryption-processor/src/main/java/uk/ac/ebi/ega/file/encryption/processor/services/EncryptService.java]: As discussed ignoring this change.

**Comment by ashutoshebi** [comment]
Done with changes. Kindly review.

**Pull request merged**